core/bin/external_node/src/config/mod.rs

```diff
@@ -158,6 +159,9 @@ pub(crate) struct OptionalENConfig {
     /// Maximum response body size in MiBs. Default is 10 MiB.
     #[serde(default = "OptionalENConfig::default_max_response_body_size_mb")]
     pub max_response_body_size_mb: usize,
+    /// Method-specific overrides in MiBs for the maximum response body size.
+    #[serde(default = "MaxResponseSizeOverrides::empty")]
```

Comment:
@tomg10 Should we leave the default overrides empty for the EN, or can we come up with a more intelligent default (e.g., `eth_getTransactionReceipt=None`)?

Reply:
Should we really do it? 
Let's disable non limit responses.